javalang/tokenizer.py

```diff
@@ -524,6 +539,9 @@ def tokenize(self):
                 # of moving try_operator higher in the chain because operators
                 # aren't as common and try_operator is expensive
                 token_type = Operator
+            elif c == ':' and c_next == ':':
```

Comment:
This branch can be removed if MethodReference is removed and '::' added to the Operator values set.